src/v/cluster/types.h

```diff
@@ -32,6 +32,12 @@ namespace cluster {
 using consensus_ptr = ss::lw_shared_ptr<raft::consensus>;
 using broker_ptr = ss::lw_shared_ptr<model::broker>;
 
+struct non_replicable_topic {
+    static constexpr int8_t current_version = 1;
```

Comment:
the version should be serialized as part of the network message, and then verified when handling. there are a bunch of examples of how to do this in cluster/types.{h,cc}

Reply:
This comment has not been resolved

Reply:
since the version is always 1, there's no need to serialize and verify